src/script/components/InputBar/InputBar.tsx

```diff
@@ -459,8 +459,15 @@ export const InputBar = ({
     };
   }, []);
 
-  const saveDraft = async (editorState: string) => {
-    await saveDraftState(storageRepository, conversation, editorState, replyMessageEntity?.id, editedMessage?.id);
+  const saveDraft = async (editorState: string, plainMessage: string) => {
+    await saveDraftState(
+      storageRepository,
+      conversation,
+      editorState,
+      plainMessage,
+      replyMessageEntity?.id,
+      editedMessage?.id,
+    );
```

Comment:
Could you refactor `saveDraftState` to take an object instead of those multiple params? It's error prone, hard to maintain

```suggestion
    await saveDraftState({
      storageRepository,
      conversation,
      editorState,
      plainMessage,
      replyMessageEntityId: replyMessageEntity?.id,
      editedMessageId: editedMessage?.id
      },
    );
```